src/storage/src/table/streaming_table/state_table.rs

```diff
@@ -59,6 +60,9 @@ pub struct StateTable<S: StateStore> {
     /// Used for serializing the primary key.
     pk_serializer: OrderedRowSerializer,
 
+    /// Used for deserializing the primary key. Debug-only for now.
+    pk_deserializer: OrderedRowDeserializer,
```

Comment:
I think `OrderedRowSerializer` and `OrderedRowDeserializer` should be merged into one later.

Currently `OrderedRowSerializer` doesn't store data types inside, but I believe this is not correct.